gootstrap.go

```diff
@@ -13,8 +13,8 @@ import (
 	"text/template"
 )
 
-//go:embed templates/*
-var templates embed.FS
+//go:embed templates/basic/*
```

Comment:
When you're embedding a directory tree with `go:embed`, entries starting with a . are ignored unless they're named explicitly. The `*` includes entries starting with a . so that works.

Reply:
It looks like the motivation for implementing it this way was to avoid the `.DS_Store` files that Mac OS tends to litter its filesystem with: https://github.com/golang/go/issues/42328
Thanks, Apple!

Reply:
auheuaheaueh I remember this because I had some issues with dotfiles not being embedded here, but I did miss it on the review, sorry (I dont remember it that well...I just thing I was bitten by this more than once :laughing: ).

Fucking Bradley/Apple !!!

Reply:
It also ignores underscore...didn't remember that :laughing: 

"all the files to embed will ignore names beginning with dot or underscore, the same as the go command does
for deciding what to build."

why the fuck underscore ?

Reply:
Not sure, maybe some editors create temporary files with names starting with underscore?